khl/client.py

```diff
@@ -363,3 +367,28 @@ async def fetch_blocked_friends(self) -> List[Friend]:
 
     async def start(self):
         await asyncio.gather(self.handle_pkg(), self.gate.run(self._pkg_queue))
+
+    async def message_view(self, msg_id: str) -> Dict:
+        """get message detail by message id"""
+        if len(msg_id) > 0:
```

Comment:
no need to check, for that the kook API will fail if msg id not valid
user is clear that the param is invalid if API fails
but nothing is retern/raised if param not pass the check